subprojects/docs/src/docs/userguide/publishingOverview.adoc

```diff
@@ -0,0 +1,262 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[publishing_overview]]
+== Publishing
+
+The vast majority of software projects build something that aims to be consumed in some way. It could be a library that other software projects use or it could be an application for end users. _Publishing_ is the process by which the thing being built is made available to consumers.
+
+In Gradle, that process looks like this:
+
+ 1. Define what to publish (_<<glossary:publication,publications>>_)
+ 2. Define where to publish it to (_<<sub:terminology_repository,repositories>>_)
+ 3. Do the publishing (via _tasks_)
```

Comment:
I need to think about this. I see your point, but this may be more of an issue of duplication. The links aren't supposed to direct the reader to an explanation of each step, but simply link to the glossary if they want to go there.

Anyway, something needs to change. I may redo lines 43-51.

Reply:
- [ ] Rewrite this section